pkg/preflight/assets/host-preflights.yaml

```
@@ -20,6 +20,35 @@ spec:
     - blockDevices:
         # rook block storage or cstor is enabled and not upgrade
         exclude: '{{ and (or (and .Installer.Spec.Rook.Version .Installer.Spec.Rook.IsBlockStorageEnabled) (and .Installer.Spec.OpenEBS.Version .Installer.Spec.OpenEBS.IsCstorEnabled)) (not .IsUpgrade) | not }}'
+    - tcpPortStatus:
+        collectorName: "Kubernetes API TCP Port Status"
+        port: 6443
+        exclude: '{{ and .IsPrimary (not .IsUpgrade) | not }}'
+    - tcpPortStatus:
+        collectorName: "ETCD Client API TCP Port Status"
+        port: 2379
+        exclude: '{{ and .IsPrimary (not .IsUpgrade) | not }}'
+    - tcpPortStatus:
+        collectorName: "ETCD Server API TCP Port Status"
+        port: 2380
+        exclude: '{{ and .IsPrimary (not .IsUpgrade) | not }}'
+    - tcpPortStatus:
+        collectorName: "Kubelet API TCP Port Status"
+        port: 10250
+        exclude: '{{ and (not .IsUpgrade) | not }}'
```

```suggestion
        exclude: '{{ .IsUpgrade }}'
```
I don't feel strongly about this. Consistency with the others is nice too.

---

ya. i was going for consistency. makes it simpler to add to this if necessary...